resource-validator/src/test/scala/com/broadinstitute/dsp/resourceValidator/mocks.scala

```diff
@@ -12,9 +14,18 @@ class FakeDbReader extends DbReader[IO] {
 
   override def getStagingBucketsToDelete: Stream[IO, BucketToRemove] = Stream.empty
 
-  override def getK8sClustersToDelete: Stream[IO, GKEModels.KubernetesClusterId] = Stream.empty
+  override def getKubernetesClustersToDelete: Stream[IO, KubernetesClusterToRemove] = Stream.empty
 
   override def getDeletedDisks: Stream[IO, Disk] = Stream.empty
 
   override def getInitBucketsToDelete: Stream[IO, InitBucketToRemove] = Stream.empty
 }
+
+class FakeGooglePublisher extends GooglePublisher[IO] {
```

Comment:
not necessarily blocking this PR..but can you add this to wb-libs

Reply:
yea - I can add as a follow-up PR - there's something else I'd like to add to wb-libs as well